modules/Bio/EnsEMBL/Production/Pipeline/Xrefs/gencode_sources.json

```diff
@@ -16,7 +16,7 @@
     {
       "name" : "UniParc",
       "parser" : "ChecksumParser",
-      "file" : "ftp://ftp.ebi.ac.uk/pub/contrib/uniparc/upidump.lis",
+      "file" : "http://ftp.ebi.ac.uk/pub/contrib/uniparc/upidump.lis",
```

Comment:
Should it be `https` instead? I tested this uri, and it's available with this protocol